assemble/conf/accumulo-env.sh

```diff
@@ -56,7 +56,7 @@ if [[ -n "$CLASSPATH" ]]; then
 else
   CLASSPATH="${conf}"
 fi
-CLASSPATH="${CLASSPATH}:${lib}/*:${HADOOP_CONF_DIR}:${ZOOKEEPER_HOME}/*:${HADOOP_HOME}/share/hadoop/client/*"
+CLASSPATH="${CLASSPATH}:${lib}/*:${lib}/ext/*:${HADOOP_CONF_DIR}:${ZOOKEEPER_HOME}/*:${HADOOP_HOME}/share/hadoop/client/*"
```

Comment:
I don't think this is necessary, because it should be handled by the dynamic class path configuration (`general.dynamic.classpaths`). If a user needs it configured on their own (static) class path, they can always add this to their own config `CLASSPATH` environment variable in their deployments.

Reply:
I agree. I will close this PR.